src/object/gtype.rs

```diff
@@ -0,0 +1,16 @@
+use types::gsize;
+
+pub type GType = gsize;
```

Comment:
All types in this module are, in fact, already defined in gobject-sys (though some of them may lack fields that are unimportant to Rust bindings).